wvd-sh/WVD scaling script/basicScale.ps1

```diff
@@ -192,61 +217,61 @@ $PeakEndDateTime = [datetime]::Parse($CurrentDateTime.ToShortDateString() + ' '
 
 
 
-        #check the calculated end time is later than begin time in case of time zone
-        if ($PeakEndDateTime -lt $PeakBeginDateTime) {
-          $PeakEndDateTime = $EndPeakDateTime.AddDays(1)
-        }
+#check the calculated end time is later than begin time in case of time zone
+if ($PeakEndDateTime -lt $PeakBeginDateTime) {
```

Comment:
No need to check the calculated end time is later than begin time in case of time zone. Please find the below pull request URL
https://github.com/Azure/RDS-Templates/commit/e4143629129d5bf734f9420c43dbe4947a409208